pheweb/run_pheweb.py

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env python3
 
+# Use this script for development to serve pheweb from a given directory instead of one installed with pip
+#
+# E.g.:
+#
+# cd /path/to/where_config.py_is
+#
+# /path/to/pheweb/pheweb/run_pheweb.py /path/to/pheweb serve --port 8080 --num-workers=4
```

Comment:
Seems a bit clunky, as now the path to pheweb has to be the second argument. In case we always have path to pheweb as a part of the run_pheweb.py's path, couldn't we get the path by getting the absolute path (with `os.path.abspath(relative_path)` and taking the parent folder from that? 
I'd also rather run the server so that the config file is given as an argument, instead of having to be in the same folder. Then we could e.g. quickly use different configs for running different revisions of the data, e.g.
```
/path/to/pheweb/run_pheweb.py /path/to/pheweb serve --config r5_conf.py --port 8080 
# OR
/path/to/pheweb/run_pheweb.py /path/to/pheweb serve --config r2_conf.py --port 8080 
```
Instead of having two different folders with two files, both named config.py, but with different contents, or swapping config files by overwriting the config.py with them.

Though this might be quite a lot of work, it seems the config file loading is a bit gnarly.